subjects/mobile-dev/quizz-app/README.md

```diff
@@ -1,32 +1,23 @@
-# Quizz App
+## Quizz App
 
-### Introduction
+### Instructions
 
-Develop a [**stateful Quizz app**](https://docs.flutter.dev/development/ui/interactive) that displays categories that you can choose questions from.
-Possible categories: history, pop culture, math, geography, etc (appropriate name, image must be added to all categories).
+You need to develop a fully functional [**stateful Quizz app**](https://docs.flutter.dev/development/ui/interactive) in Flutter that allows users to select a **category** and answer **true or false questions** related to the selected category.
 
-After tapping on the category the new route must show a corresponding image to the category, the question itself, true/false buttons.
+**Features:**
 
-After the question has been answered, you must let the user know if the question was answered correctly.
+- The app needs to feature at least 5 different categories that the users can select from the main page, i.e, history, culture, math, geography, etc.
+- The app needs to feature at least at least 10 questions to each `quizz`.
+- Appropriate name and images must be added to all of the selected categories. (follow the examples bellow)
```

Comment:
My grammar checker suggest to change `all of the` to `all the`

Reply:
done